ribs/emitters/_gradient_operator_emitter.py

```diff
@@ -0,0 +1,339 @@
+"""Provides the GradientOperatorEmitter."""
+
+import numpy as np
+
+from ribs._utils import check_1d_shape, check_batch_shape, validate_batch_args
+from ribs.emitters._emitter_base import EmitterBase
+
+
+class GradientOperatorEmitter(EmitterBase):
+    """Generates solutions with a gradient arborescence, with coefficients
+    parameterized by a fixed Gaussian distribution.
+
+    This emitter is from `Fontaine 2021 <https://arxiv.org/abs/2106.03894>`_.
+    It proceeds in two stages. The first stage samples a batch of intermediate
+    solutions from the archive and (optionally) applies Gaussian perturbation
```

Comment:
Also mention the iso line operation here and how it can be used instead of Gaussian.